styles/globals.css

```diff
@@ -66,4 +66,50 @@ footer{
 .NotFound a:hover{
   color: rgb(20, 245, 196);
   text-shadow: 1px 1px 2px rgb(159, 252, 232), 0 0 2em;  
+}
+
+.userDiv{
+  padding: 10px;
+}
+.userWelcom{
+
+  color: white;
+}
+.welcomeA{
+  padding-left: 10px;
+  color: yellow;
+}
+.userLink{
+  
+  padding: 10px;
+  color: brown;
+}
+
+.userDiv a:visited{
+  text-decoration: none;
+  color: brown;
+}
+
+.loginUsername{
+  padding: 5px;
+  font-size: 20px;
+}
+.loginUsername input{
+  position: relative;
+  left: 10px;
+}
+.loginPassword{
+  
+  padding: 20px;
+  font-size: 20px;
+
+}
+.loginPassword input{
+  
+  position: relative;
+  left: 5px;
+}
+
+.loginSubmit{
+  font-size: 15px;
```

Comment:
For better separation of concern one can use css modules instead of using the global css file. This isolates your styling for your page/component. Benefits includes no classname overrides, prossible merge conflict avoidable, easy to find rules if editing is needed on page/component level.

Reply:
Agree

Reply:
i agree too.  That also binds the  entirety of the rest of the project togheter.